### ISSUE TITLE
Index issue when filtering the channels of interest.

### ISSUE THREAD

**Issue by KathrineSN** [PINE-Lab/HAPPE#9]
Hi HAPPE support team

I am currently trying to run the HAPPE_v4.m pipeline on an EEGLAB file of resting state low density EEG-data. However, I keep encountering an index error, as it won't allow me to include more than 1 channel (see the errorlog below)
![image](https://github.com/PINE-Lab/HAPPE/assets/46521720/d9530131-28ed-4806-a2bc-3461d61a0706)
It seems the error comes from line 525 in the HAPPE_v4.m script
![image](https://github.com/PINE-Lab/HAPPE/assets/46521720/48d5a164-cb4b-40ce-bca3-26651b2e59f8)
and the 'chanIDs' which is causing problems is a 1x32 cell with the channel names
![image](https://github.com/PINE-Lab/HAPPE/assets/46521720/cd480561-fbca-4921-a84d-ee3aa2954799)
These are my parameters settings
![image](https://github.com/PINE-Lab/HAPPE/assets/46521720/2c46c26c-4225-422a-a775-3e1b7f22f33a).

I have checked my EEGLAB file and it looks fine. I also only have the EEGLAB version installed that is in the HAPPE-master folder.
My MATLAB version is: R2022b Update 4 (9.13.0.2166757)

Do you have any idea what might be causing this error?

Thank you in advance!

**Comment by PINE-Lab** [comment]
Hi Kathrine! Could you please confirm that your data includes channel locations? For instance, when you load the file into EEGLAB, do your channels have labels?

**Comment by KathrineSN** [comment]
Yes, when I load the data in EEGLAB I do have the channel locations
![image](https://github.com/PINE-Lab/HAPPE/assets/46521720/ce96217e-cb77-410f-90ce-50179199ca1e)
I can also see the labels when I view the raw data
